src/watchwindow.cpp

```diff
@@ -0,0 +1,219 @@
+#include "watchwindow.h"
+
+#include <QLabel>
+#include <QVBoxLayout>
+#include <QDebug>
```

Comment:
Most of them are not necessary. While inspecting this i also noticed extra including in .h file. Thanks.